osmaxx-py/osmaxx/tests/test_utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+from django.test import TestCase
+import os
+from osmaxx.utils import PrivateSystemStorage
+
+
+class PrivateSystemStorageTestCase(TestCase):
+    def setUp(self):
+        random_directory_name_length = 10
+        self.random_directory_name = ''.join(
+            random.choice(string.ascii_lowercase + string.digits) for _ in range(random_directory_name_length)
+        )
+
+    def test_creates_a_new_directory_if_it_does_not_exist(self):
+        random_directory_path = os.path.join('/tmp/', self.random_directory_name)
```

Comment:
Yeah, I think I'd prefer an unlikely-to-clash sensible.

The book 'Clean Code' lists
> Don't be arbitrary

as a smell/heuristic (Chapter 17, Section 'General', **G32**).